### ISSUE TITLE
Do more work on Beginner-friendly page

### ISSUE THREAD

**Issue by cryptodamsky** [kodadot/docs#42]
Let’s get the page to the state where we explain what NFTs are, as it is one of use cases of KodaDot.

Ref: https://github.com/kodadot/nft-gallery/issues/192

**Comment by thea-exe** [comment]
How do you feel about putting that in the README instead because we're introducing KodaDot as an NFT marketplace- might as well explain what an NFT is?

**Comment by thea-exe** [comment]
[https://github.com/kodadot/docs/issues/2](https://github.com/kodadot/docs/issues/2)

**Comment by thea-exe** [comment]
on it

**Comment by cryptodamsky** [comment]
Let’s have bulletpoints on what we would want to add there?

**Comment by thea-exe** [comment]
Yep! I forgot to push it up and it was in my branch the whole time D: will
ping you in the morning :)
